yoti-stub/src/services/YotiRequestProcessor.ts

```diff
@@ -121,17 +129,34 @@ export class YotiRequestProcessor {
 	async getSession(sessionId: string): Promise<Response> {
 
 		const lastUuidChars = sessionId.slice(-4);
+
+		const positiveScenario = (lastUuidChars.substring(0,3) === '000');
 		this.logger.info({ message: "last 4 ID chars", lastUuidChars});
+		this.logger.info({ message: "is a positive scenario", positiveScenario});
+		let mediaId;
 
+		if (positiveScenario) {
+			switch(lastUuidChars) {
+				case '0000': // UK Passport
```

Comment:
please update the mapping for these in https://govukverify.atlassian.net/wiki/spaces/FTFCRI/pages/3537600538/F2F+Yoti+Stub+Design+Scenarios

Reply:
Done